region/maridia/inner-pink.json

```diff
@@ -3240,6 +3398,21 @@
                   "name": "Base",
                   "notable": false,
                   "requires": [ "Morph" ]
+                },
+                {
+                  "name": "G-Mode Morph Exit Tunnel Before Leaving G-Mode",
+                  "notable": false,
+                  "requires": [ {"obstaclesCleared": ["B"]} ]
```

Comment:
The strats which don't clear the obstacle don't appear to have a way out.  
4->5->1 is still useful for avoiding a tricky jump.

4->5->1 also sounds very not safe with the puyos

Reply:
Right, that is the only option for indirect mode. The only way out would be if the item is Morph